packages/server-core/src/updateAppConfig.ts

```diff
@@ -141,13 +139,21 @@ export const updateAppConfig = async (): Promise<void> => {
 
   const chargebeeSettingPromise = knexClient
     .select()
-    .from<ChargebeeSettingType>(chargebeeSettingPath)
-    .then(([dbChargebee]) => {
-      if (dbChargebee) {
-        appConfig.chargebee = {
-          ...appConfig.chargebee,
-          ...dbChargebee
-        }
+    .from<EngineSettingType>(engineSettingPath)
+    .whereIn('key', [EngineSettings.Chargebee.Url, EngineSettings.Chargebee.ApiKey])
+    .then((dbChargebeeSettings) => {
+      const chargebeeSettings = dbChargebeeSettings.reduce(
+        (acc, setting) => {
+          acc[setting.key] = setting.value
+          return acc
+        },
+        {} as Record<string, string>
+      )
+
+      appConfig.chargebee = {
+        ...appConfig.chargebee,
+        url: chargebeeSettings[EngineSettings.Chargebee.Url],
+        apiKey: chargebeeSettings[EngineSettings.Chargebee.ApiKey]
       }
     })
```

Comment:
This needs to be done in https://github.com/ir-engine/ir-engine/blob/5213e54f39c914040a3f739c49d2883531f96dc5/packages/server-core/src/setting/engine-setting/engine-setting.ts#L62

Reply:
@hanzlamateen done.

Reply:
`chargebeeSettingPromise` needs to be remove from updateAppConfig.ts